### PR TITLE
Skeleton REST implementation

### PR DESCRIPTION
**Pull request by ajaynair**
Added skeleton code for the REST implementation
All the calls responds with correct but static data

### CODE REVIEW

File: docs/rest_spec.txt [hunk 1]
```diff
@@ -0,0 +1,70 @@
+REST specification 
+
+An email is compared to cattle and the analogy is used throughout. Sometimes, stretched to the point of it being nauseating! Blame Omkar Ekbote for it.
```
Comment (dotbugfix) [inline docs/rest_spec.txt:3]: How professional of you!

File: docs/rest_spec.txt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  Test: curl -i http://localhost:5000/api/fodder/ingredients
+HTTP/1.0 200 OK
+Content-Type: application/json
+Content-Length: 148
+ContentType: application/json
+Access-Control-Allow-Origin: *
+Server: Werkzeug/0.14.1 Python/2.7.5
+Date: Thu, 08 Nov 2018 12:49:30 GMT
+
+["S.No","Name of Child","Class","Sponsor","Reference","Sponsor Mail Id","Reference Mail Idemail subject","email body","attachment","has_sent_mail"]
```
Comment (dotbugfix) [inline docs/rest_spec.txt]: Is this a copy-paste error or did 2 heads get smashed into each other?

File: docs/rest_spec.txt [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  Test: curl -i http://localhost:5000/api/vomit
+HTTP/1.0 200 OK
+Content-Type: application/json
+Content-Length: 587
+ContentType: application/json
+Access-Control-Allow-Origin: *
+Server: Werkzeug/0.14.1 Python/2.7.5
+Date: Thu, 08 Nov 2018 12:46:36 GMT
+
+{"data":[["1","Avani KulkarniNur","Amit Shah","Sumit Kumar","amit.shah@gmail.comsumit.kumar@gmail.comReport cardHello Amit,\nPlease see attached the report card\nThanks,\nSneh Foundation\navani.kulkarni.pdf",false],["2","Bhavani ShahSenior","Amar Shinde","Sunil krishna","amar.shinde@gmail.comsunilkrishna@gmail.comReport cardHello Amar,\nPlease see attached the report card\nThanks,\nSneh Foundation\nbhavani.shah.pdf",false]],"headers":["S.No","Name of Child","Class","Sponsor","Reference","Sponsor Mail Id","Reference Mail Idemail subject","email body","attachment","has_sent_mail"]}
```
Comment (dotbugfix) [inline docs/rest_spec.txt]: Hmm, let me check what format the DataGrid GUI element wants and I might tell you to change this.

Also, the values look weird.

File: docs/rest_spec.txt (hunk 2 above)
Comment (ajaynair) [reply docs/rest_spec.txt]: This issue is due to an error in py file. Have resolved it.

File: docs/rest_spec.txt (hunk 3 above)
Comment (ajaynair) [reply docs/rest_spec.txt]: The values won't be weird with the latest push.

**Review by dotbugfix — APPROVED** [review]

**Pull request merged**